src/main/java/com/lazerycode/jmeter/mojo/AbstractJMeterMojo.java

```diff
@@ -271,4 +278,12 @@ protected void loadMavenProxy() {
 
 	}
 
```

Comment:
I think this should stay in ConfigureJMeterMojo.java.  That way our configuration code is in the Configure MOJO.